app/src/main/res/values/strings.xml

```diff
@@ -58,7 +59,7 @@
     <string name="references_added">References Added</string>
     <string name="outreach_programs_and_events_dashboard">Outreach Programs and Events Dashboard</string>
     <string name="menu_settings">Settings</string>
-    <string name="not_enrolled">You are not enrolled in any course</string>
+    <string name="not_enrolled">Not enrolled in any course</string>
```

Comment:
Unnecessary change.

Reply:
Got it. I returned the previous content of this string.